lib/response/headers.ex

```diff
@@ -1,5 +1,8 @@
 defmodule HTTPServer.Response.Headers do
+  alias HTTPServer.Request
```

Comment:
It may just be the formatting, but I find this whole file hard to read.

Reply:
I put the build parameter definitions on one line and I think that makes it slightly better.